packages/calling/calling/src/CallingClient/CallingClient.test.ts

```diff
@@ -1416,7 +1416,6 @@ describe('CallingClient Tests', () => {
       const restoreSpy = jest.spyOn(callingClient as any, 'restorePreviousRegistration');
       const restartRegSpy = jest.spyOn(callingClient as any, 'restartRegistration');
       const reconnectSpy = jest.spyOn(callingClient as any, 'reconnectOnFailure');
-      const logSpy = jest.spyOn(log, 'log');
```

Comment:
Why has this been removed? I don't see the log removed from the code

Reply:
we have logSpy globally present too, we don't need to spy again and this was causing eslint error